src/xdocs/config_javadoc.xml

```diff
@@ -1302,16 +1302,44 @@ public class TestClass {
 &lt;module name="JavadocStyle"/&gt;
         </source>
 
+        <p>Example:</p>
+        <source>
+public class Test {
+    &#47;**
+    * Some description here. // OK
+    *&#47;
```

Comment:
2 lines above should have indentation = 5

Reply:
Done